packages/framework/src/Twig/Javascript/JavascriptCompilerService.php

```diff
@@ -148,7 +148,7 @@ private function getPathWithTimestamp($relativePath, $timestamp)
     private function getRelativeTargetPath($javascript)
     {
         $relativeTargetPath = null;
-        if (strpos($javascript, 'admin/') === 0 || strpos($javascript, 'frontend/') === 0 || strpos($javascript, 'common/') === 0) {
+        if (strpos($javascript, 'admin/') === 0 || strpos($javascript, 'frontend/') === 0 || strpos($javascript, 'common/') === 0 || strpos($javascript, 'custom_') === 0) {
```

Comment:
You are using directory "custom_admin" everywhere but here there is checking for "custom_". Why?

Reply:
When I was discovering how it works, I added a line `custom_admin/*.js` into twig, and nothing happend. So I was debugging, and found this condition. I have no idea why the condition is here and why is so strict, it just restricted me from adding the javascript.

So, this is not as strict as previous ones and can be used for example for "custom_frontend"

Reply:
Still not answered, why there is checking for "custom_" and not for "custom_admin"?